### DISCUSSION TITLE
工具

### DISCUSSION THREAD

**Discussion by giscus** [youngjunning/blog.zisheng.pro#85] [Announcements]
# 工具

工欲善其事必先利其器

https://youngjuning.js.org/tool/

**Comment by youngjunning** [comment]
- [Raycast](https://www.raycast.com/)：Raycast 是一个速度极快、完全可扩展的启动器。它可以让您完成任务、计算、共享公共链接等等。
- [Online UUID Generator Tool](https://www.uuidgenerator.net/)
